tests/algorithms/test_partition_selection.py

```diff
@@ -89,7 +89,7 @@ def test_laplace_keep_and_return_noised_value(
         ],
         [
             (10, 1, 1e-5, 1, 0.017845473615190732),
-            (1100, 0.2, 1e-7, 10, 0.007884076914531857),
+            (1100, 0.2, 1e-7, 10, 1.0),
```

Comment:
is there a reason to change this number?

Reply:
Yes, I made a fix in the partition selection library. It's utility fix, not privacy, from privacy stand point everything was correct, but the utility suffered. So it's expected.